akka/shopping-cart-scala/project/cinnamon.sbt

```diff
@@ -1,2 +1,2 @@
 // The Cinnamon Telemetry plugin
-addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % "2.17.3")
+addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % "2.19.0")
```

Comment:
Need to bump all the Akka dependencies in the Scala sample. Sync with the changes in https://github.com/akka/akka-guide/pull/856.